taccsite_cms/templates/assets_site.html

```diff
@@ -21,7 +21,7 @@
 
 
 
-{# Portal & Docs #}
+{# "CEPv2" Portal & Frontera Docs #}
```

Comment:
Maybe a bit more context in the comment?

`{# Impacts the clients "CEPv2" Portal & Frontera Docs #}`

Reply:
Agreed. Updated.